src/listener.h

```diff
@@ -49,13 +50,15 @@ struct Listener {
     SLIST_ENTRY(Listener) entries;
 };
 
+static int parse_boolean(char *boolean);
 
```

Comment:
static function declarations belong in the .c file since they are not exported beyond the module (akin to private methods in Java).